_includes/navbar-w-links.html

```diff
@@ -0,0 +1,52 @@
+<!-- NAV -->
+
+<nav class="navbar" aria-label="main">
+	<button aria-label="open" onClick="toggleMenu()" class="menu-icon"><img src="/images/hamburger-open-menu.svg" alt="open menu" aria-hidden="true"></button>
```

Comment:
Just an FYI, if there is an `aria-hidden`, the `alt` will not be read (so the `alt` isn't really necessary here)